src/plugins/safeSnap/constants.ts

```diff
@@ -101,6 +101,2590 @@ export const ORACLE_ABI = [
   'function withdraw() public'
 ];
 
+export const UMA_MODULE_ABI = [
```

Comment:
I recommend to use the compact string ABI format to reduce space and import it as a JSON file, here is a sample script to convert the ABI:
https://github.com/snapshot-labs/snapshot.js/blob/master/test/abi.ts